media-gfx/zbar/zbar-0.23.ebuild

```diff
@@ -0,0 +1,203 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit autotools flag-o-matic java-pkg-opt-2 multilib-minimal python-single-r1 virtualx
+
+DESCRIPTION="Library and tools for reading barcodes from images or video"
+HOMEPAGE="https://github.com/mchehab/zbar"
+SRC_URI="https://linuxtv.org/downloads/${PN}/${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+IUSE="dbus graphicsmagick gtk gtk2 imagemagick introspection java jpeg png python qt5 static-libs test +threads v4l X xv"
+REQUIRED_USE="
+	introspection? (
+		|| ( gtk gtk2 )
+	)
+	python? ( ${PYTHON_REQUIRED_USE} )
+	qt5? ( X )
+	test? (
+		|| ( graphicsmagick imagemagick )
+		${PYTHON_REQUIRED_USE}
```

Comment:
wouldn't it rather depend on `python`?

Reply:
no https://github.com/mchehab/zbar/blob/1d51925e6cc9151f4a73781989f21fcd7b57ef32/test/Makefile.am.inc#L114